.github/workflows/report.yml

```diff
@@ -48,32 +46,54 @@ jobs:
     name: resolve dependencies
     runs-on: ubuntu-latest
     outputs:
-      pull_request_number: ${{ steps.pr.outputs.pull_request_number }}
-      allow_simplecov_action: ${{ github.ref_name == 'master' || steps.pr.outputs.pull_request_number != '' }}
+      all_changed_files: ${{ steps.changed-files.outputs.all_modified_files }}
     steps:
       - uses: actions/checkout@v2
+      - name: get changed files
+        id: changed-files
+        uses: tj-actions/changed-files@v1.1.2
       - uses: pinnacles/common-cicd-actions/.github/actions/setup-lang@v0.1.0
         with:
-          use_ruby: ${{ inputs.use_ruby }}
+          use_ruby: 'true'
```

Comment:
always use_ruby is true because of the report.yml only be used for report of rspec and simplecov.